examples/references-count/src/locations/EntryList.ts

```diff
@@ -5,7 +5,8 @@ import {
 } from '@contentful/app-sdk';
 import { CollectionProp, ContentTypeProps, PlainClientAPI } from 'contentful-management';
 
-const DEFAULT_ENTRY_FIELD_LOCALE = 'en-US';
+
+const DEFAULT_ENTRY_FIELD_LOCALE = 'en-US'; // consider changing to the default locale of your space
```

Comment:
Can we just use `sdk.locales.default` instead?

Reply:
YES! I didn't know we have it